src/paired_delimiter.py

```diff
@@ -23,26 +42,70 @@ class PairedDelimiter:
     PairedDelimiter("diamond", "angleBrackets", "<", ">"),
     PairedDelimiter("escaped quad", "escapedDoubleQuotes", '\\"', '\\"'),
     PairedDelimiter("escaped twin", "escapedSingleQuotes", "\\'", "\\'"),
+    PairedDelimiter("escaped round", "escapedParentheses", "\\(", "\\)"),
     PairedDelimiter("quad", "doubleQuotes", '"', '"'),
     PairedDelimiter("round", "parentheses", "(", ")"),
     PairedDelimiter("skis", "backtickQuotes", "`", "`"),
-    PairedDelimiter("void", "whitespace", " ", " "),
+    PairedDelimiter("void", "whitespace", " ", " ", is_selectable=False),
```

Comment:
We will implement this one differently.  We'll create a scope type called "paint" because it's not really a proper surrounding pair